core/src/main/res/values/strings.xml

```diff
@@ -527,6 +527,12 @@
     <string name="back_button_go_to_page_title">Select page</string>
     <string name="pref_delete_removes_from_queue_title">Delete removes from Queue</string>
     <string name="pref_delete_removes_from_queue_sum">Automatically remove an episode from the queue when it is deleted.</string>
+    <string name="pref_filter_feed_title">Subscription Filter</string>
+    <string name="pref_filter_feed_sum">Filter your subscriptions screen</string>
+    <string name="hide_subscriptions_where_counter_is_zero">Hide if counter is zero</string>
+    <string name="no_filter_label">None</string>
+    <string name="feed_is_filtered">Your subscriptions are filtered.</string>
+    <string name="subscriptions">Subscriptions</string>
```

Comment:
Another string with the same meaning (`subscriptions_label`) already exists. Please use that one instead to ensure consistency (in contrast to the `None` string, this both refers to the same thing)